telemetry/specs/telemetry_report.mdk

```diff
@@ -550,158 +653,178 @@ particular deployment.
-    
-      -  Copy the INT Hop ML to the Telemetry Report MD Length, copy the INT DS Instruction 
-         to the Telemetry Report DSMdBits field, and pad the Variable Optional Baseline & 
-         DS Metadata with the special all-ones reserved value out to MD Length, or 
-         
-      - Set the Telemetry Report DSMdBits field to zero and rederive the Telemetry Report 
-         MD Length from RepMdBits.
-    
+  : Bitmap that indicates which optional domain specific metadata is
+    present in the record of the telemetry report. Each bit represents
+    4 octets or a multiple of 4 octets of domain specific optional metadata.
+    When using INT-MD or INT-MX, if the Domain Specific ID does not match
+    any Domain ID known to this node, then the node may either:
+
+      -  Copy the INT *Hop ML* to the Telemetry Report *MD Length*, copy the INT *DS Instruction*
+         to the Telemetry Report *DSMdBits* field, and pad the Variable Optional Baseline &
+         DS Metadata with the special all-ones reserved value out to *MD Length*, or
+
+      - Set the Telemetry Report *DSMdBits* field to zero and rederive the Telemetry Report
+         *MD Length* from *RepMdBits*.
+
 * DSMdstatus (16b): Domain Specific Md Status
-  : Bitmap that indicates the domain specific metadata status. 
-  
+  : Indicates the domain specific metadata status.
+
 * Variable Optional Baseline Metadata
-  : The metadata corresponding to the RepMdBits, 4 octets for each bit, except 8 octets 
+  : The metadata corresponding to the *RepMdBits*, 4 octets for each bit, except 8 octets
     for bits 4, 5 & 6.
-    
-* Variable Optional Domain Specific Metadata
-  : The metadata corresponding to the DSMdBits, 4 octets for each bit.
-    
-* Truncated Packet
-  : L2/L3/ESP/L4 of the packet for flow details 
-  
-* Additional DS Extension Data
-  : Additional Domain Specific Specific Extension Data not associated with DSMdBits.
-  
-* TLV
-  : Type Length Value format. Multiple TLV formated data following the Variable Optional
-  Baseline Metadata and Variable Optional Domain Specific Metadata.
-
- 0                   1                   2                   3
- 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|TLVType|  Rsvd |   TLVLength   |        TLV Data Template      | 
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|          Variable TLV Data as identified by TLVType           |                      
-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
-TLVType (4b): TLV Type – TLV data type.
-0 – Domain Specific Extension Data
-1 - Ethernet
-2 – IPv4
-3 – IPv6
-4-15 – Reserved.
+* Variable Optional Domain Specific Metadata
+  : The metadata corresponding to the *DSMdBits*, 4 octets or a multiple of
+    4 octets for each bit.
+
+### Inner Report Contents for *InType* 1 (*TLV*) (4+ octets) { #tlv }
```

Comment:
Inner Report Contents: 
Another term used to indicate "Telemetry Report Contents" or "Individual Report Contents". Can we unify them?

Reply:
"Inner Report Contents" is *not* the same as "Telemetry Report Contents".

How about:
"Telemetry Report Contents" => "Individual Report Primary Contents" or "Individual Report Main Contents" or "Individual Report Principal Contents"
"Inner Report Contents" => "Individual Report Inner Contents"

Reply:
that's right.

I like this:
"Telemetry Report Contents" => "Individual Report Main Contents"
"Inner Report Contents" => "Individual Report Inner Contents"

Reply:
As a 2nd thought, the term "inner" seems confusing because we have "Telemetry Reports Outer Header" up there. The outer header is for the entire coalesced report pkt, and the inner header is for each individual report. The outer vs inner are applied to two different objects.

One idea is to change
"Telemetry Reports Outer Header" => "Telemetry Report Group Header", 
where a group may have one or more individual reports in it. 
We can define the term 'group' first and use it else where.

Reply:
DOne